### PR TITLE
(blog) Status update 2020-07-03

### PR DESCRIPTION
**Pull request by daianamezdrea**

### CODE REVIEW

File: _posts/2020-07-3-status-update.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+title: "3th July 2020: IAC Team Status Update"
+author: daianamezdrea
+categories:
+  - team
+  - status
+tags:
+  - releases
+  - community_management
+  - java_ks
+  - tagmail
```
Comment (sanfrancrisko) [inline _posts/2020-07-3-status-update.md:11]:
```suggestion
  - tagmail
  - apache
```

File: _posts/2020-07-3-status-update.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  - shoutouts
+---
+
+The IAC team has been working on a lot of things over the past week.
+We want to share with our community some of the most interesting things.
+Here’s a brief overview of what we’ve been up to:
+
+## Module Releases from IAC Team
+
+- [puppetlabs/java_ks](https://github.com/puppetlabs/puppetlabs-java_ks)(3.2.0)
```
Comment (sanfrancrisko) [inline _posts/2020-07-3-status-update.md:23]:
```suggestion
- [puppetlabs/apache](https://github.com/puppetlabs/puppetlabs-apache)(5.5.0)
- [puppetlabs/java_ks](https://github.com/puppetlabs/puppetlabs-java_ks)(3.2.0)
```

File: _posts/2020-07-3-status-update.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+[lswith](https://github.com/lswith) - Ignore stderr of serverspec commands by setting request pty to false [309](https://github.com/puppetlabs/puppet_litmus/pull/309)
+
+## Community Shout outs for IAC
+
+We think that is no doubt about our community contributors, they are always ready to help and they have excellent ideas. Here are some highlights of this week:
+
+Thanks to [Guillaume Leroy](https://github.com/leroyguillaume) for the bug [fix](https://github.com/puppetlabs/puppetlabs-tomcat/pull/400) on tomcat module.
+
+Thanks to [Marc Schöchlin](https://github.com/scoopex) for the kubernetes PRs which bring a lot of new changes.
+
```
Comment (sanfrancrisko) [inline _posts/2020-07-3-status-update.md:43]: Sorry, my bad. I should have piped up with some contributions:
```suggestion
We'd also like to call out two fixes in the `puppetlabs-apache` module: [h-haaks](https://github.com/h-haaks) for [2041](https://github.com/puppetlabs/puppetlabs-apache/pull/2041) and [farebers](https://github.com/farebers) for [2021](https://github.com/puppetlabs/puppetlabs-apache/pull/2021).
```

File: _posts/2020-07-3-status-update.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+## Customer tickets closed this week
+
+The team have closed out the following customer tickets: MySQL issue (MODULES-10597) and a vcsrepo issue (MODULES-10659).
```
Comment (sanfrancrisko) [inline _posts/2020-07-3-status-update.md:46]:
```suggestion
The team have closed out the following customer tickets: MySQL issue ([MODULES-10597](https://tickets.puppetlabs.com/browse/MODULES-10597)) and a vcsrepo issue ([MODULES-10659](https://tickets.puppetlabs.com/browse/MODULES-10659)).
```

File: _posts/2020-07-3-status-update.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+*FYI my contribution has been released. It's fantastic! Faster than I ever thought!* - [Filip Hrebek](https://github.com/fhrbek)
+
+*I'm happy that my (little) contribution to @puppetize #tomcat module was merged* - [Francis](https://github.com/leroyguillaume)
+
+Thank you, folks! Thank you for your contributions and your positive feedback!
+
+## Final exam for Daiana 
+
+Tuesday, Daiana had her last exam from college, now she is officially an engineer and she started the work on docker and kubernetes module.
```
Comment (sanfrancrisko) [inline _posts/2020-07-3-status-update.md:60]:
```suggestion
Tuesday, Daiana had her last exam from college, now she is officially an engineer and she started the work on the docker and kubernetes modules.
```

File: _posts/2020-07-3-status-update.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+## Apache module work in progress 
+
+This week sees a lot of excellent community contribution’s in `v5.5.0` of the [puppetlabs-apache](
```
Comment (sanfrancrisko) [inline _posts/2020-07-3-status-update.md:64]: Oops - sorry, my bad:
```suggestion
This week sees a lot of excellent community contributions in `v5.5.0` of the [puppetlabs-apache](
```

File: _posts/2020-07-3-status-update.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+https://forge.puppet.com/puppetlabs/apache) module.
+We have also been working on improving our test strategy around [Apache MODs](https://httpd.apache.org/docs/2.4/mod) to ensure we can turn around updates to the module in a faster manner.
+Check out [sanfrancrisko’s](https://github.com/sanfrancrisko) [blog post]({% _posts/2020-07-03-apache-mod-test-improvements.md %}) which outlines what we’ve done and how we’re handling test coverage for [Apache MODs](https://httpd.apache.org/docs/2.4/mod), going forward.
```
Comment (sanfrancrisko) [inline _posts/2020-07-3-status-update.md:67]: Another screw up on my side! Sorry!
```suggestion
Check out [sanfrancrisko’s](https://github.com/sanfrancrisko) [blog post]({% link _posts/2020-07-03-apache-mod-test-improvements.md %}) which outlines what we’ve done and how we’re handling test coverage for [Apache MODs](https://httpd.apache.org/docs/2.4/mod), going forward.
```

File: _posts/2020-07-3-status-update.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+We have released v0.18.4 of the [puppet_litmus gem][https://github.com/puppetlabs/puppet_litmus].
+There were quite a number of fixes but a few highlights worth mentioning:
+
+[DavidS](https://github.com/DavidS) - Protect version reporting from undefined-ness: [312](https://github.com/puppetlabs/puppet_litmus/pull/312) 
+
+[lswith](https://github.com/lswith) - Ignore stderr of serverspec commands by setting request pty to false [309](https://github.com/puppetlabs/puppet_litmus/pull/309)
+
+## Community Shout outs for IAC
+
+We think that is no doubt about our community contributors, they are always ready to help and they have excellent ideas. Here are some highlights of this week:
```
Comment (carabasdaniel) [inline _posts/2020-07-3-status-update.md:38]:
```suggestion
We appreciate our community contributors as they are always ready to help and they have excellent ideas. Here are some highlights of this week:
```

File: _posts/2020-07-3-status-update.md (hunk 4 above)
Comment (daianamezdrea) [reply _posts/2020-07-3-status-update.md:46]: My bad here, I forgot to add the links ... Daiana--

**Comment by sanfrancrisko** [comment]
When https://github.com/puppetlabs/iac/pull/85 is shipped and published, re-kick the tests and then publish this if it goes green

**Pull request closed without merging**